tests/core/mempool/test_mempool_manager.py

```diff
@@ -329,3 +329,54 @@ async def test_sb_twice_with_eligible_coin_and_different_spends_order() -> None:
     assert result == (expected_cost, MempoolInclusionStatus.PENDING, Err.MEMPOOL_CONFLICT)
     assert mempool_manager.get_spendbundle(sb_name) == sb
     assert mempool_manager.get_spendbundle(reordered_sb_name) is None
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "opcode,lock_value,expected_status,expected_error",
+    [
+        (ConditionOpcode.ASSERT_SECONDS_RELATIVE, -2, MempoolInclusionStatus.SUCCESS, None),
+        (ConditionOpcode.ASSERT_SECONDS_RELATIVE, -1, MempoolInclusionStatus.SUCCESS, None),
+        (ConditionOpcode.ASSERT_SECONDS_RELATIVE, 0, MempoolInclusionStatus.SUCCESS, None),
```

Comment:
I think the discrepancy between `SECONDS_RELATIVE` and `HEIGHT_RELATIVE` warrants a comment, to remind a reviewer of this test what the rules are

Reply:
Done.

Reply:
the `ASSERT_SECONDS_RELATIVE` condition requires the block timestamp to be *greater than* the coin timestamp + this argument. meaning that 0 still won't accept being spent in the same block.